src/main/java/wolox/training/models/Book.java

```diff
@@ -0,0 +1,149 @@
+package wolox.training.models;
+
+import java.util.Objects;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
+@Entity
+public class Book {
+
+    private String genre;
```

Comment:
Add the @Column annotation in this field

Reply:
In this case, the gender field is not required and the column name is not different. Therefore, is not required to add the **@column** annotation